buku

```diff
@@ -950,29 +949,34 @@ class BukuDb:
         except IndexError:
             logger.error('Index out of bound')
 
-    def export_bookmark(self, fp):
+    def export_bookmark(self, file_path, export_tags=None):
         '''Export bookmarks to a Firefox
         bookmarks formatted html file.
 
-        Params: Path to file to export to
+        Params: file_path: Path to file to export to
+        export_tags: tag to export, None for all
         '''
```

Comment:
Please leave the newline after function desc. We follow the convention in `Buku`.